tests/shared/src/test/scala/matryoshka/data/fix.scala

```diff
@@ -20,20 +20,18 @@ import matryoshka.exp.Exp
 import matryoshka.helpers._
 import matryoshka.patterns.CoEnv
 import matryoshka.scalacheck.arbitrary._
-import matryoshka.specs2.scalacheck._
+import matryoshka.scalacheck.cogen._
 
 import scala.Int
 
+import org.scalacheck.Test
 import org.specs2.mutable._
 import scalaz._, Scalaz._
 import scalaz.scalacheck.ScalazProperties._
 
-class FixSpec extends Specification with CheckAll with AlgebraChecks {
-  "Fix" should {
-    "satisfy relevant laws" in {
-      checkAll(equal.laws[Fix[Exp]])
-    }
-  }
+class FixSpec extends Specification with AlgebraChecks {
+
+  equal.laws[Fix[Exp]].check(Test.Parameters.default)
```

Comment:
I’m not very happy with this approach. It seems to work (prints out `+ equal.transitive: OK, passed 100 tests.`), but looks (check the build output) like it’s executed outside of the spec.